docs/guides/cli/20_proxy-and-tunnel.mdx

```diff
@@ -0,0 +1,376 @@
+  "nbf": 1627560064,
+  "session": {
+    "id": "b73d7dc4-f565-4fea-951e-8c23ee05783f",
+    "active": true,
+    "expires_at": "2021-07-30T12:01:02.96663Z",
+    "authenticated_at": "2021-07-29T12:01:03.024365Z",
+    "issued_at": "2021-07-29T12:01:02.966652Z",
+    "identity": {
+      "id": "0f0c9bec-6b68-47f3-b62b-7750f065359c",
+      "schema_id": "default",
+      "schema_url": "https://<project-slug>.projects.oryapis.com/api/kratos/public/schemas/default",
+      "state": "active",
+      "state_changed_at": "2021-07-29T12:01:02.844472Z",
+      "traits": {
+        "email": "dev+docs@ory.sh",
+        "firstname": "Ory Docs",
+        "vegeterian": true
+      },
+      "verifiable_addresses": [
+        {
+          "id": "64b3f270-70fc-48b8-8704-8085c783362b",
+          "value": "dev+docs@ory.sh",
+          "verified": false,
+          "via": "email",
+          "status": "sent",
+          "verified_at": null,
+          "created_at": "2021-07-29T12:01:02.888562Z",
+          "updated_at": "2021-07-29T12:01:02.888562Z"
+        }
+      ],
+      "recovery_addresses": [
+        {
+          "id": "526a2b25-2b9a-445b-982d-97862d9bbc9b",
+          "value": "dev+docs@ory.sh",
+          "via": "email",
+          "created_at": "2021-07-29T12:01:02.897407Z",
+          "updated_at": "2021-07-29T12:01:02.897407Z"
+        }
+      ],
+      "created_at": "2021-07-29T12:01:02.877417Z",
+      "updated_at": "2021-07-29T12:01:02.877417Z"
+    }
+  },
+  "sub": "0f0c9bec-6b68-47f3-b62b-7750f065359c"
+}
+```
+
+
+### Usage in Local Development
+
+When using the Proxy for local development, point to the URL where your application runs. 
+
+For example, if your application is available at `http://localhost:3000`, run this command: 
+
+```shell
+ory proxy http://localhost:3000
+```
+
+By default, the Proxy creates an entry point at `http://localhost:4000`. To get access to Ory endpoints, 
+you must access the app through the proxy using This URL instead of the actual address on which your
+application is running. 
+
+If you want to adjust the entry point URL, pass the desired address along with the application URL, 
+for example: 
+
+```shell
+ory proxy http://localhost:3000 http://localhost:3001
+```
+
+### Usage in Production
+
+You can use Ory Proxy in a production setup when you are self-hosting Ory components. 
+To do that, set the entry point URL to the domain where you want to expose the app through the Ory Proxy: 
+
+```shell
+ory proxy http://localhost:3000 https://my-domain.com
+```
+
+### Running on Virtual and Dedicated Servers
+
+To use the Proxy with an application deployed on a VM or a Dedicated Server, 
+run the application server and the Proxy in parallel.
+
+For example, if you are running a NodeJS server on port `3000` and you expose web
+traffic on port `8080`: 
+
+```shell
+ory proxy --port 8080 http://127.0.0.1:3000 https://your-domain.com &
+
+# This should be your server command
+node your-entrypoint.js
+```
```

Comment:
Questionable - always have something like NGINX or Traefik in front of your deployment. The proxy should be exposed directly, this makes it difficult to do maintenance and load balancing. The proxy does not offer the same functionality as NGINX / Traefik etc. 

You can still run this command as is since this will rewrite all cookies etc to match `https://your-domain.com`. There is also a flag that you can set for which domain the cookies should be set on if you are running this on a subdomain `https://sub.your-domain.com`. 

```shell
ory proxy --cookie-domain example.org \
		http://127.0.0.1:3000 \
		https://ory.example.org
```

Reply:
You then need to point NGINX / Traefik to your proxy url `http://localhost:4000` unless you changed the port with `--port 3001`.

Reply:
I understand what you mean with the need for something like NGINX, but this is just a simple example of how you could use the proxy if you really wanted to. Wouldn't it work if you looked at it like that?

Reply:
I think just note here that you would run your ingress proxy in front of the ory proxy (such as NGINX etc)

Reply:
done